Source/Core/UICommon/GameFile.cpp

```diff
@@ -323,9 +314,7 @@ std::string GameFile::GetUniqueIdentifier() const
   if (GetRevision() != 0)
     info.push_back("Revision " + std::to_string(GetRevision()));
 
-  std::string name(GetLongName(lang));
-  if (name.empty())
-    name = GetName();
+  const std::string& name = GetName();
```

Comment:
This won't use the long name anymore AFAICT -- is this intended?

Reply:
This has the same behavior as before. GetName will return the long name unless you pass in `false` (or unless there is no long name, in which case it returns the filename).